src/containers/AccountManagement.jsx

```diff
@@ -53,10 +54,30 @@ export default class AccountManagement extends Component {
       })
   }
 
+  validInfos (input) {
+    const { t } = this.context
+    // Check if field is empty or not
+    if (input.target.value !== '') {
```

Comment:
The Alerter system provide like an API to be called from outside of the component rendering. But this component should be redesigned in order to be more universal and usable from a library like cozy-ui.